src/Documentation/DocumentationGenerator/Extensions.cs

```diff
@@ -179,11 +179,37 @@ TypedExpression input
             // are an empty set; in that case, we want an empty string.
             ?? string.Empty;
 
-        internal static string ToSyntax(this QsCallable callable, Action<IRewriteStep.Diagnostic>? onDiagnostic = null) =>
-            SyntaxTreeToQsharp.DeclarationSignature(
+        internal static string ToSyntax(this QsFunctor functor) =>
+            functor.Tag switch
+            {
+                QsFunctor.Tags.Adjoint => "Adj",
+                QsFunctor.Tags.Controlled => "Ctl",
+                _ => "__invalid__"
+            };
+
+        internal static string ToSyntax(this QsCallable callable, Action<IRewriteStep.Diagnostic>? onDiagnostic = null)
```

Comment:
`onDiagnostic` is an unused parameter?

Reply:
Good catch, thank you; it used to be used, but that was stripped out in response to one of @bettinaheim's earlier points of feedback; must have missed a case.